src/webserver/HttpBase.h

```diff
@@ -242,12 +242,16 @@ class HttpBase
                 },
                 dosGuard_.isWhiteListed(*ip)))
         {
+            // Connection getting rate limited due to full connection queue
             http::response<http::string_body> res{
-                http::status::ok, req_.version()};
-            res.set(http::field::server, "clio-server-v0.0.0");
+                http::status::service_unavailable, req_.version()};
```

Comment:
I don't think this one is supposed to be `service_unavailable`. This is not the same as rate limiting, this is just a normal error.